eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -185,40 +178,32 @@ public QueryResponse getAnnotations(@PathVariable("variantCoreString") String va
             variantEntity = getVariantByCoordinatesAndAnnotationVersion(variantCoreString, annotationVepVersion,
                     annotationVepCacheVersion);
         } catch (AnnotationMetadataNotFoundException | IllegalArgumentException ex) {
-            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-            return setQueryResponse(ex.getMessage());
+            return new ResponseEntity(ex.getMessage(), HttpStatus.BAD_REQUEST);
         }
 
-        List<Annotation> annotations = new ArrayList<>();
         if (!variantEntity.isPresent() || variantEntity.get().getAnnotation() == null) {
-            response.setStatus(HttpServletResponse.SC_NOT_FOUND);
-            return setQueryResponse(buildQueryResult(annotations, 0));
+            return new ResponseEntity(null, HttpStatus.NOT_FOUND);
         }
+        Link coreVariantLink = new Link(linkTo(methodOn(VariantWSServerV2.class).getCoreInfo(variantCoreString,
+                species, assembly, response)).toUri().toString(), "CoreVariant");
```

Comment:
try to keep the case consistent with every other endpoint, let's use camelCase
```suggestion
                species, assembly, response)).toUri().toString(), "coreVariant");
```